recipes/doctr-versions-menu/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "doctr-versions-menu" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2713d6923639c9a8e96894b340d5e0780c2c923c3d785ec94a2c6196e9d4fe45 
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - doctr-versions-menu = doctr_versions_menu.cli:main
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    - setuptools
+
+  run:
+    - python >=3.5
+    - click >=6.7
+    - configobj >=5.0.6
+    - doctr
+    - packaging
+    - pyparsing
+    - sphinx
+
+test:
+  imports:
+    - doctr_versions_menu
+
+  commands:
+    - doctr-versions-menu --help
+
+about:
+  home: https://github.com/goerz/doctr_versions_menu
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Sphinx extension and command to add a versions menu to Doctr-deployed documentation'
+  doc_url: https://simplejson.readthedocs.io/
```

Comment:
This `doc_url` should be corrected or removed.

Reply:
Good catch! Addressed via https://github.com/conda-forge/staged-recipes/pull/12250/commits/1f000c20219b59e174dc8bc0e336720c6728c53d